tests/main.fmf

```diff
@@ -4,3 +4,7 @@ contact: Petr Šplíchal <psplicha@redhat.com>
 tier: 2
 tag: [container, virtual]
 require: [tmt]
+
+adjust:
+  - check: avc
```

Comment:
I'd recommend making `check` a list right away. There might be tests that will add more checks in the future, and it's ugly to decode "cannot add list and dict" error message spitted out by fmf :/

Reply:
Makes sense, fixed in 06da5fda.